pages/securing.rst

```diff
@@ -4,6 +4,12 @@
 Securing Graylog
 ****************
 
+To secure your Graylog Setup, you should not use one of our pre-configured Images, create your own unique installation where you understand each component and secure the environment by design. Expose only the Services that are needed and secure them whenever possible with TLS/SSL and some kind of authentification. Do not use the pre-created appliances as critical production environments. 
```

Comment:
* setup
  * images
  * understand each component and security is build-in by design (?)
  * services
  * authentication
  * for critical production environments